mongodbatlas/data_source_mongodbatlas_third_party_integration_test.go

```diff
@@ -243,7 +239,8 @@ func testAccCreateThirdPartyIntegrationConfig() *matlas.ThirdPartyIntegration {
 func testGenString(length int, charSet string) string {
 	sequence := make([]byte, length)
 	for i := range sequence {
-		sequence[i] = charSet[seeder.Intn(len(charSet))]
+		n, _ := rand.Int(rand.Reader, big.NewInt(int64(len(charSet))))
```

Comment:
[nit] can we move `big.NewInt(int64(len(charSet)))` outside the loop given is the same for every iteration?

Reply:
I imagined the compiler would be smart enough to do that automatically (plus this is test code), but I made the change anyway.